styles/styles.css

```diff
@@ -0,0 +1,348 @@
+body {
+  margin: 0;
+  width: 100%;
+}
+
+/*
+header {
+  background-color: #0A3C61;
+  height: 21rem;
+  text-align: left;
+  display: flex;
+  justify-content: center;
+}
+*/
+header {
+  background-color: #0A3C61;
+  height: 24rem;
+  justify-content: flex-start;
+
+}
+
+header h1 {
+  color: black;
+  position: absolute;
+  left: 13rem;
+  top: 1rem;
+  font-size: 60px;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 25px;
+  font-family: sans-serif;
+  font-style: italic;
+  line-height: 36px;
+}
+
+.b-box {
+  width: 30rem;
+  height: 100px;
+}
+
+/*
+.header-img {
+  object-fit: cover;
+  width: 100%;
+  height: inherit;
+  object-position: top;
+}
+*/
+
+.header-img {
+  width: 100%;
+  height: 24rem;
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.ftrlogo {
+  height: 30%;
+}
+
+.b2box {
+  height: 70%;
+  width: 40%;
+  margin: 4%;
+  min-width: 227px;
+  max-width: 484px;
+}
+
+.hdrtxt {
+  justify-content: center;
+}
+
+.ckout {
+  margin: 4%;
+}
+
+.art1 {
+  width: 60%;
+  display: inline-block;
+  margin: 10%;
+}
+
+
+
+
+
+
+/* Style the links inside the navigation bar */
+nav {
+  font-family: sans-serif;
+}
+
+nav li {
+  padding: 0 5px 0 5px;
+  position: relative;
+  left: 11rem;
+}
+
+nav li a:after{
+  content: " / ";
+  position: absolute;
+  right: 0;
+  top : 15px;
+}
+
+nav li:last-child a:after{

```

Comment:
If you want to challenge yourself with these pseudo elements. See how you could leverage :not